src/settings/LocationLibraries.js

```diff
@@ -107,10 +110,14 @@ class LocationLibraries extends React.Component {
 
   onChangeInstitution = (e) => {
     this.setState({ institutionId: e.target.value, campusId: null });
+
+    sessionStorage.setItem('institutionIdLibraries', e.target.value);
   }
 
   onChangeCampus = (e) => {
     this.setState({ campusId: e.target.value });
```

Comment:
`e.target.value` can be moved to variable

Reply:
done